blas/gonum/level3cmplx128.go

```diff
@@ -306,15 +306,17 @@ func (Implementation) Zhemm(side blas.Side, uplo blas.Uplo, m, n int, alpha comp
 		return
 	}
 
-	if alpha == 0 {
-		if beta == 0 {
-			for i := 0; i < m; i++ {
-				ci := c[i*ldc : i*ldc+n]
-				for j := range ci {
-					ci[j] = 0
-				}
+	if beta == 0 {
```

Comment:
Can you please summarize what is the essence of the change? If alpha is zero, the behavior doesn't change, for any value of beta the function sets C and returns. For non-zero alpha and zero beta C is first zeroed out but then overwritten with alpha*A*B so the zeroing out was unnecessary? However, the issues has been fixed, so I must be missing something?